src/test/java/jpa/com/jaenyeong/domain/station/StationRepositoryTest.java

```diff
@@ -0,0 +1,100 @@
+package jpa.com.jaenyeong.domain.station;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@DataJpaTest
+@DisplayName("Station Repository 테스트")
+class StationRepositoryTest {
+    @Autowired
+    private StationRepository stations;
+
+    private Station gangnam;
+    private Station jamsil;
+    private Station kkachisan;
+
+    @BeforeEach
+    void setUp() {
+        gangnam = new Station("강남역");
```

Comment:
```suggestion
        강남 = new Station("강남역");
```

테스트 코드니까 가독성을 위해 변수명을 한글로 써도 괜찮을 것 같습니다